### ISSUE TITLE
Pylint should not report literal-comparison when testing for tuple

### ISSUE THREAD

**Issue by chaoflow** [PyCQA/pylint#3031]
### Steps to reproduce
```python
def foo(bar=()):
    if bar is not ():
        ...
```

### Current behavior

pylint reports "literal-comparison"

### Expected behavior

pylint should regard the "is (not)" comparison correctly.

### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.7.3 (default, Apr  3 2019, 19:16:38) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
```

**Comment by PCManticore** [comment]
Yeah, we can probably exempt tuples from the check.

**Comment by chaoflow** [comment]
@PCManticore I think it's noteworthy that only starting with Python 3.7 ``(1,) is (1,)`` before this was only true for the empty tuple.

**Comment by chaoflow** [comment]
Following up on this, I think only the empty tuple should be exempted.

compare

```python
tuple(range(0)) is ()
tuple(range(1)) is (0,)
```
